examples/proof-of-concept/gcp-iot/gcp-iot-core-connector/gcp-iot-core-connector.iml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
You shouldn't need to add this to the repo. It's generated by IntelliJ. I've added `*.iml` to my `.gitignore_global` file.